Controller/Checkout/EstimateShipping.php

```diff
@@ -101,11 +101,11 @@ public function execute()
                 )
             );
             $response = ['error' => 1,
-                         'text'  => 'Internal error'];
+                         'message'  => 'There was an error calculating the available shipping methods'];
```

Comment:
yeah on the frontend we call `alert(response.message);` with that text, same with the estimate tax as well

Reply:
I'm not sure that we want to show this error message to the customer. It's helpful for debugging purposes and it's not a big deal that the customer can see the specific error, but what is the customer's course of action? I think any messaging we surface to the customer should be clear about what their next steps are.

Reply:
yeah that's a good point, I wanted to be able to distinguish from when shipping or tax failed for Sentry purposes, so I'll still return the same value here, but then I'll change the error displayed to the customer to be more generic, since I don't think we can provide a generic path forward in these cases, since it could have messed up for a bunch of different reasons I think

Reply:
ok that makes sense, thanks!